app/views/items/index.html.erb

```diff
@@ -76,4 +67,14 @@
         <% end %>
       </tbody>
     </table>
+    <%= form_tag(item_search_path, :method => "get") do %>  
```

Comment:
does changing this break tests?

Reply:
what do you mean?

Reply:
just was checking it didnt break any tests